common/common.go

```diff
@@ -144,10 +145,11 @@ func DefaultNetwork(repoPath string, opts ...NetOption) (NetBoostrapper, error)
 }
 
 type NetConfig struct {
-	HostAddr    ma.Multiaddr
-	ConnManager cconnmgr.ConnManager
-	Debug       bool
-	GRPCOptions []grpc.ServerOption
+	HostAddr       ma.Multiaddr
+	ConnManager    cconnmgr.ConnManager
+	Debug          bool
+	GRPCOptions    []grpc.ServerOption
+	PubSubDisabled bool
```

Comment:
To go along with the comment below, this would just be `PubSub`.

Reply:
Fixed